MobileCenter.podspec

```diff
@@ -16,59 +16,66 @@ Pod::Spec.new do |s|
                       Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
 
                       3. Mobile Center Distribute:
-                      Mobile Center Distribute provides the capability to display in-app updates to your app users when a new version of the application is released.
+                      Mobile Center Distribute provides the capability to display in-app updates to your app users when a new version of the application is released. Not available for macOS SDK.
```

Comment:
I am wondering if there is a way to make the "MacOS unavailability" more visible?